test/hotspot/jtreg/TEST.quick-groups

```diff
@@ -969,7 +969,6 @@ vmTestbase_nsk_jvmti_quick = \
   vmTestbase/nsk/jvmti/GetMethodName/methname003/TestDescription.java \
   vmTestbase/nsk/jvmti/GetObjectMonitorUsage/objmonusage001/TestDescription.java \
   vmTestbase/nsk/jvmti/GetObjectMonitorUsage/objmonusage002/TestDescription.java \
-  vmTestbase/nsk/jvmti/GetObjectMonitorUsage/objmonusage003/TestDescription.java \
```

Comment:
This file needs a copyright year update.

Reply:
Thanks, fixed now.